inc/saiport.h

```diff
@@ -2104,6 +2104,44 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_SUPPORTED_LINK_TRAINING_MODE,
 
+    /**
+     * @brief Enable ARS on port. This enables the control for quality measure in switch pipeline.
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_ARS_ENABLE,
+
```

Comment:
Does it refer to ingress or egress port ?

Reply:
ARS is always egress attribute. SAI pipeline to not reference to an object processing as ingress or egress.